src/core/replicators.ts

```diff
@@ -0,0 +1,186 @@
+import * as tt from '../../typings/telegram-types'
+import Markup from '../markup'
+const { formatHTML } = Markup
+
+type Replicate<X, K extends keyof (tt.Message & X)> = Required<
+  Pick<tt.Message & X, K>
+>
```

Comment:
This magic is the last thing I don't like. I want to use the `Message` types after my next PR to `typegram`.

Forgot to mention, wait with releasing `typegram` until the next PR is settled.